Header header
string name
float32 confidence
geometry_msgs/Point position
geometry_msgs/Point[] contour_pts 
sensor_msgs/CompressedImage image